docs/content/learn/tutorials/general/extension.md

```diff
@@ -0,0 +1,770 @@
+    "import-profile": "...",
+    "system-characteristics": "...",
+    "system-implementation": "..."
+  }
+}
+{{< /highlight >}}
+
+A new GLC property was created by setting the `prop` object's `name` property to "glc".  The acronym "glc" could be used by other organizations (and have a completely different meaning) so a namespace `ns` was set.  Lastly, GLCs have many data attributes including territory, country codes, state codes, county codes, city codes, and duty station codes, so this example defined a single "glc" `prop` but used `class` to provide context for the property's set `value`.
+
+Props are prevalent throughout OSCAL models appearing not only in `metadata` and `back-matter` objects, but in the majority of OSCAL data items.  Regardless of which OSCAL property is being extended, the approach is consistent, as described in this section of the tutorial.
+{{% /tab %}}
+{{% tab %}}
+{{< highlight yaml "linenos=table" >}}
+system-security-plan:
+  uuid: ce16b9af-6853-4abe-9e27-b79d034c0adc
+  metadata:
+    title: OSCAL Basic Prop Example
+    last-modified: '2022-01-01T09:30:00-005'
+    version: 20220531
+    oscal-version: 1.0.0
+    locations:
+    - title: ...
+      props:
+      - name: type
+        value: security-operations-center
+        class: regional
+      - name: glc
+        ns: http://example.com/ns/oscal
+        value: '11'
+        class: state-code
+      - name: glc
+        ns: http://example.com/ns/oscal
+        value: '0010'
+        class: city-code
+      - name: glc
+        ns: http://example.com/ns/oscal
+        value: '840'
+        class: country-code
+  import-profile: ...
+  system-characteristics: ...
+  system-implementation: ...
+{{< /highlight >}}
+
+A new GLC property was created by setting the `prop` object's `name` property to "glc".  The acronym "glc" could be used by other organizations (and have a completely different meaning) so a namespace `ns` was set.  Lastly, GLCs have many data attributes including territory, country codes, state codes, county codes, city codes, and duty station codes, so this example defined a single "glc" `prop` but used `class` to provide context for the property's set `value`.
+
+Props are prevalent throughout OSCAL models appearing not only in `metadata` and `back-matter` objects, but in the majority of OSCAL data items.  Regardless of which OSCAL property is being extended, the approach is consistent, as described in this section of the tutorial.
+{{% /tab %}}
+{{% /tabs %}}
+
+## Links
+
+Links in OSCAL provide a means to reference an arbitrary resource, which allows local or remote content to be referenced. Links are available to use on many of the key fields in the OSCAL models.
+
+A link can:
+
+1. Reference (external) information that is not represented in OSCAL format.  This could include references to (cybersecurity) laws and regulations, references to organizational standards and guides, references to software bill of materials (SBOM), and more.
```

Comment:
```suggestion
1. Reference (external) information that is not represented in OSCAL format.  This could include references to (cybersecurity) laws and regulations, references to organizational standards and guides, references to a software bill of materials (SBOM), and more.
```